packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/FGMClusterManagersController.h

```diff
@@ -0,0 +1,70 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Flutter/Flutter.h>
+#import <GoogleMaps/GoogleMaps.h>
+@import GoogleMapsUtils;
+
+#import "messages.g.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+// Defines cluster managers controller interface which
+// is responsible for adding/removing/returning one or more cluster managers.
+@interface FGMClusterManagersController : NSObject
+
+/// Initializes FLTClusterManagersController.
+///
+/// @param callbackHandler A callback handler.
+/// @param mapView A map view that will be used to display clustered markers.
+- (instancetype)initWithMapView:(GMSMapView *)mapView
+                callbackHandler:(FGMMapsCallbackApi *)callbackHandler;
+
+/// Creates ClusterManagers and initializes them form JSON data.
+///
+/// @param clusterManagersToAdd List of clustermanager object data.
+- (void)addJSONClusterManagers:(NSArray<NSDictionary *> *)clusterManagersToAdd;
+
+/// Creates ClusterManagers and initializes them.
+///
+/// @param clusterManagersToAdd List of clustermanager object data.
+- (void)addClusterManagers:(NSArray<FGMPlatformClusterManager *> *)clusterManagersToAdd;
+
+/// Removes requested ClusterManagers from the controller.
+///
+/// @param identifiers List of clusterManagerIds to remove.
+
+- (void)removeClusterManagersWithIdentifiers:(NSArray<NSString *> *)identifiers;
+
+/// Returns the ClusterManager for the given identifier.
+///
+/// @param identifier identifier of the ClusterManager.
+/// @return GMUClusterManager if found; otherwise, nil.
```

Comment:
The comment does not need to say what the return type is; the code already does that.